builtin/providers/scaleway/resource_scaleway_ip.go

```diff
@@ -0,0 +1,66 @@
+package scaleway
+
+import "github.com/hashicorp/terraform/helper/schema"
+
+func resourceScalewayIP() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceScalewayIPCreate,
+		Read:   resourceScalewayIPRead,
+		Update: resourceScalewayIPUpdate,
+		Delete: resourceScalewayIPDelete,
+		Schema: map[string]*schema.Schema{
+			"server": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"ip": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func resourceScalewayIPCreate(d *schema.ResourceData, m interface{}) error {
+	scaleway := m.(*Client).scaleway
+	resp, err := scaleway.NewIP()
+	if err != nil {
+		return err
+	}
+
+	d.SetId(resp.IP.ID)
+	return resourceScalewayIPUpdate(d, m)
+}
+
+func resourceScalewayIPRead(d *schema.ResourceData, m interface{}) error {
+	scaleway := m.(*Client).scaleway
+	resp, err := scaleway.GetIP(d.Id())
+	if err != nil {
```

Comment:
Can we handle the issue when someone deletes an IP from the console / cli? We should do something like

```
if err != nil {
    if resp.StatusCode == 404 {
        d.SetId("")
        log.Printf("[DEBUG] IP Not Found so refreshing from state")
        return nil
    }
}
```